bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/admin/StoreAdminHandler.java

```diff
@@ -477,4 +490,85 @@ public Response getRemoteByUrl( final @PathParam( "packageType" ) String package
         return response;
     }
 
+
+    @ApiOperation( "Revalidation of Artifacts Stored on demand" )
+    @ApiResponses( { @ApiResponse( code = 200, response = ArtifactStore.class, message = "Revalidation for Remote Repositories was successfull" ),
+        @ApiResponse( code = 404, message = "Revalidation is not successfull" ), } )
+    @Path( "/revalidate" )
```

Comment:
You'll need to change the @PathParam to `packageType` to match what's in the class-level annotation.